frontends/mit-open/webpack.config.js

```diff
@@ -84,6 +84,19 @@ const getWebpackConfig = ({ mode, analyzeBundle }) => {
           env: { NODE_ENV: JSON.stringify(mode) },
         },
       }),
+      new webpack.DefinePlugin({
+        APP_SETTINGS: {
+          embedlyKey: JSON.stringify(process.env.EMBEDLY_KEY),
+          ocw_next_base_url: JSON.stringify(process.env.OCW_BASE_URL),
```

Comment:
You can remove `ocw_next_base_url`. It was used in https://github.com/mitodl/open-discussions but is not used in MIT Open.